manala.proftpd/tests/0200_configs.goss.yml

```diff
@@ -0,0 +1,42 @@
+---
+
+file:
+  /etc/proftpd/conf.d/virtual_host_foo.conf:
+    exists: true
+    mode: "0644"
+    owner: root
+    group: root
+    filetype: file
+    contains:
+      - <VirtualHost ftp.foo.com>
```

Comment:
Maybe you can test indentation too, something like `"    ServerName ..."` instead of just `ServerName`